packages/theming/theme-types/src/Shadow.types.ts

```diff
@@ -0,0 +1,51 @@
+import { ColorValue } from 'react-native';
+
+/**
+ * Base structure of a shadow style, which has x and y offset, blur amount, and color specified
+ */
+export interface ShadowStyle {
+  x: number;
+  y: number;
+  blur: number;
+  color: ColorValue;
+}
```

Comment:
Now that I've plated around with this a little more, the naming here kinda conflicts with the type  [ShadowStyleIOS](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4f4f4a891a67881be36fa0c1d41d809f60afcdbb/types/react-native/index.d.ts#L3498), which is the platform specific iOS (and apparently macOS and win32) props the final View needs. 

It kinda feels like this should be called ShadowToken.. except that is what we call line 18. Maybe call line 18 ShadowSet? Not sure if the name "ShadowStyle" is also inherited from the pipeline or not.

Reply:
We can rename these, it makes sense not to clash with ShadowStyleIOS

Reply:
I went with `ShadowValue` as the new name which aligns with the naming scheme we have for `Variant`. I wanted to keep `ShadowToken` as is because a shadow token is specified with both ambient and key parts, so I think it's appropriate for the Token to be the type which encompasses both.